src/app/app.component.ts

```diff
@@ -9,15 +8,9 @@ import { ErrorsService } from './errors.service';
 export class AppComponent {
   teste: string = ''
 
-  constructor(private errorsService:ErrorsService) {  }
+  constructor() {  }
   title = 'angularProject';
-  errors: string[] = [];
 
   ngOnInit() { }
 
-  reciverFeedback(receivedResponse) {
```

Comment:
decide se vai usar ponto e virgula ou não, faltou tipagem nas variaveis do metodo e no proprio metodo...
```suggestion
receiverFeedback(receivedResponse: TYPE): RETURN_TYPE {
```